handlers/handlers.go

```diff
@@ -118,5 +83,33 @@ func HasContentType(r *http.Request, mimetype string) bool {
 			return true
 		}
 	}
+
 	return false
 }
+
+func init() {
+	files, err := schemasDir.ReadDir("schemas")
+
+	if err != nil {
+		glog.Fatal(err.Error())
+	}
+
+	for _, file := range files {
+		schemaContent, err := schemasDir.ReadFile("schemas/" + file.Name())
+
+		if err != nil {
+			glog.Fatal(err.Error())
+		}
+
+		schemaLoader := gojsonschema.NewStringLoader(string(schemaContent))
+		schema, err := gojsonschema.NewSchema(schemaLoader)
+
+		if err != nil {
+			glog.Fatal(err.Error())
+		}
+
+		schemaName := strings.TrimSuffix(file.Name(), ".json")
```

Comment:
To be more generic (e.g. yaml)
```suggestion
		schemaName := strings.TrimSuffix(file.Name(), filepath.Ext(file.Name()))
```